cmd/derpprobe/derpprobe.go

```diff
@@ -29,6 +29,7 @@ var (
 	tlsInterval  = flag.Duration("tls-interval", 15*time.Second, "TLS probe interval")
 	bwInterval   = flag.Duration("bw-interval", 0, "bandwidth probe interval (0 = no bandwidth probing)")
 	bwSize       = flag.Int64("bw-probe-size-bytes", 1_000_000, "bandwidth probe size")
+	bwTUNAddress = flag.String("bw-tun-addr", "", "if specified, bandwidth probes will be performed over a TUN device at this address in order to simulate real clients. We will use a /30 subnet including this IP address.")
```

Comment:
only IPv4 addresses are valid, worth adding in flag doc and/or flag name

`in order to exercise TCP-in-TCP in similar fashion to TCP over Tailscale via DERP`